src/Pos/Util/JsonLog.hs

```diff
@@ -177,3 +177,6 @@ jsonLogDefault x = do
     JL.jsonLogDefault jlc x
 
 deriving instance CanJsonLog (t m) => CanJsonLog (TaggedTrans tag t m)
+
+-- Required for @Explorer@ @BListener@ redirect
+deriving instance CanJsonLog m => CanJsonLog (TaggedTrans tag IdentityT m)
```

Comment:
It looks quite strange. Doesn't it conflict with the previous line? Anyway, if it compiles, I don't mind leaving it here.